### PR TITLE
Add missing breaking change

### PR DESCRIPTION
**Pull request by amyblais**
For https://community.mattermost.com/core/pl/h8pbhwozkfyb9yuus63wqac38w

### CODE REVIEW

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Latest Mattermost Releases:
 ### Important Upgrade Notes
  - Removed the deprecated Insights feature.
  - Mattermost Boards and various other plugins have transitioned to being fully community supported. See this [forum post](https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669) for more details.
+ - The ``channel_viewed`` websocket was changed to ``multiple_channels_viewed``.
```
Comment (amyblais) [inline source/install/self-managed-changelog.md:22]: Let me know what other details might be needed here (e.g. who/what does this impact).

Comment (lieut-data) [reply source/install/self-managed-changelog.md:22]:
```suggestion
 - The ``channel_viewed`` websocket event was changed to ``multiple_channels_viewed``, and is now only triggered for channels that actually have unread messages.
```

Comment (lieut-data) [reply source/install/self-managed-changelog.md:22]: cc @larkox, is this an accurate summary?

Comment (larkox) [reply source/install/self-managed-changelog.md:22]: That seems right, yes.

**Review by larkox — APPROVED** [review]

**Pull request merged**